src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -26,10 +29,19 @@ def validate(self, data):
             ))
 
 
+class SchemaField(serializers.WritableField):
+    def validate(self, data):
+        try:
+            validate(data)
+        except SchemaValidationError as e:
+            raise ValidationError(u'{}'.format(e.message))
```

Comment:
Can we just do `raise ValidationError(e.message)` here? We're not really formatting anything, per se.